.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '12', '10', '8', '6' ]
+        node-version: [10.x, 12.x, 13.x]
```

Comment:
why this change? The line after this load this value, so it should be updated as well.

Reply:
Not clear what you mean.
v6 was failing and you asked for actions config to be updated: https://github.com/fastify/fastify-cookie/pull/67#issuecomment-614158920